src/main/java/frc/robot/subsystems/pivot/PivotIOSparkMax.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot.subsystems.pivot;
+
+import static frc.robot.Constants.PivotConstants.*;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.trajectory.ExponentialProfile;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+
+public class PivotIOSparkMax implements PivotIO {
+  private boolean openLoop = false;
+  private double appliedVolts = 0.0;
+  private final CANSparkMax leftMotor =
+      new CANSparkMax(LEFT_MOTOR_PORT, CANSparkLowLevel.MotorType.kBrushless);
+  private final CANSparkMax rightMotor =
```

Comment:
I'm pretty sure you want one motor to follow the other motor for the pivot. You don't want to bend a shaft!